tests/loggers/test_tensorboard_logger.py

```diff
@@ -0,0 +1,26 @@
+# Copyright 2022 MosaicML Composer authors
+# SPDX-License-Identifier: Apache-2.0
+
+from unittest.mock import Mock
+
+import pytest
+
+from composer.loggers import TensorboardLogger
+from tests.fixtures import dummy_fixtures
+
+# To satisfy pyright.
+dummy_state = dummy_fixtures.dummy_state
+composer_trainer_hparams = dummy_fixtures.composer_trainer_hparams
+
+
+def test_tensorboard_logger_trainer(monkeypatch: pytest.MonkeyPatch, composer_trainer_hparams):
```

Comment:
Curious if we need this test? I think we already have coverage for this via https://github.com/mosaicml/composer/blob/16622c9457874f4f3811287fe933b0ec58107fc7/tests/callbacks/test_callbacks.py#L42 and https://github.com/mosaicml/composer/blob/16622c9457874f4f3811287fe933b0ec58107fc7/tests/callbacks/test_callback_hparams.py#L19

Reply:
oh ok didn't see this test. ok removed